### ISSUE TITLE
change menu

### ISSUE THREAD

**Issue by Tiberiu13** [ranjithprabhuk/Angular-Bootstrap-Dashboard#8]
Hello,

It is possible to change menu without reload data?

Thanks!

**Comment by phantomD92** [comment]
Yeah, it is possible to change menus dynamically in an Angular application without reloading the data using features like data binding and component architecture.
